### PR TITLE
doc: Add timescaledb config options

### PR DESCRIPTION
**Pull request by benolayinka**
#### Summary

Closes https://github.com/TheThingsIndustries/lorawan-stack-support/issues/521

@neoaggelos do these options only configure an underlying TimescaleDB database? Is there a description somewhere of what values they accept and what they do?

#### Screenshots

Bildschirmfoto 2021-08-30 um 15 06 10

#### Checklist

- [x] Scope: The referenced issue is addressed, there are no unrelated changes.
- [x] Run Locally: Verified that the docs build using `make server`, posted screenshots, verified external links. Test with `HUGO_PARAMS_SEARCH_ENABLED=true` if style changes will affect the search bar.
- [ ] New Features Marked: Documentation for new features is marked using the `new-in-version` shortcode, according to the guidelines in [CONTRIBUTING](CONTRIBUTING.md).
- [ ] Style Guidelines: Documentation obeys style guidelines in [CONTRIBUTING](CONTRIBUTING.md).
- [x] Commits: Commit messages follow guidelines in [CONTRIBUTING](CONTRIBUTING.md), there are no fixup commits left.

### CODE REVIEW

File: doc/content/integrations/storage/configuration.md [hunk 1]
```diff
@@ -25,6 +25,16 @@ Initialize the configured database with:
 $ ttn-lw-stack storage-db init
 ```
 
+{{< note >}}
+If you are using TimescaleDB, the following configuration options are also available:
+
+```
+--timescaledb.chunk-time-interval
+--timescaledb.enable-retention-policy
```
Comment (neoaggelos) [inline doc/content/integrations/storage/configuration.md:33]: Options are `--timescaledb.enable-retention-policy` and `--timescaledb.enable-retention-policy=false` (to disable).

The retention period is configured with the `--timescaledb.retention-days` flag (see below).

File: doc/content/integrations/storage/configuration.md [hunk 2]
```diff
@@ -25,6 +25,16 @@ Initialize the configured database with:
 $ ttn-lw-stack storage-db init
 ```
 
+{{< note >}}
+If you are using TimescaleDB, the following configuration options are also available:
+
+```
+--timescaledb.chunk-time-interval
```
Comment (neoaggelos) [inline doc/content/integrations/storage/configuration.md:32]: This is used to configured the chunk time interval in the TimescaleDB hypertable. Perhaps good to link to the TimescaleDB documentation for this. https://docs.timescale.com/timescaledb/latest/how-to-guides/hypertables/best-practices/#time-intervals

The default value is set to 1 week, and people will typically not need to change this unless they really know what they are doing.

Comment (neoaggelos) [reply doc/content/integrations/storage/configuration.md:32]: Note that this is a duration, so it needs to be set as `168h` (168 hours) etc, or in general `Xh` for X hours.

File: doc/content/integrations/storage/configuration.md (hunk 1 above)
Comment (neoaggelos) [reply doc/content/integrations/storage/configuration.md:33]: Note that enabling this flag will have an immediate effect and start deleting upstream messages that are older than the specified number of days.

File: doc/content/integrations/storage/configuration.md [hunk 3]
```diff
@@ -25,6 +25,16 @@ Initialize the configured database with:
...
+--timescaledb.enable-retention-policy
+--timescaledb.retention-days
```
Comment (neoaggelos) [inline doc/content/integrations/storage/configuration.md:34]: This is only effective if the `--timescaledb.enable-retention-policy` flag is enabled as well.

This is the number of days for which received messages will be stored (based on the timestamp of the upstream message). The default value for this flag is `30`, which means that messages are stored for a month.

**Review by neoaggelos — APPROVED** [review]
Indeed, all flags apply for TimescaleDB only. If a simple PostgreSQL instance is used instead, they will not have any effect.

Maybe worth noting that setting the retention policy requires TimescaleDB version 1.6 or newer.

**Review by adriansmares — APPROVED** [review]

**Pull request merged**